swarm/pss/pss.go

```diff
@@ -463,14 +463,11 @@ func (p *Pss) process(pssmsg *PssMsg, raw bool, prox bool) error {
 		payload = recvmsg.Payload
 	}
 
-	if len(pssmsg.To) < addressLength {
-		if err := p.enqueue(pssmsg); err != nil {
-			return err
-		}
+	if len(pssmsg.To) < addressLength || prox {
```

Comment:
you can submit this fix separately if you want quick result :) the rest will take a bit...

Reply:
I warmly encourage this, too.